ml_service/pipelines/diabetes_regression_build_train_pipeline.py

```diff
@@ -31,14 +31,10 @@ def main():
         print("aml_compute:")
         print(aml_compute)
 
-    run_config = RunConfiguration(conda_dependencies=CondaDependencies.create(
-        conda_packages=['numpy', 'pandas',
-                        'scikit-learn', 'tensorflow', 'keras'],
-        pip_packages=['azure', 'azureml-core',
-                      'azure-storage',
-                      'azure-storage-blob',
-                      'azureml-dataprep'])
-    )
+    # Create a reusable run configuration environment
+    conda_deps_file = "diabetes_regression/training/training_dependencies.yml"
```

Comment:
not sure if this path is going to work on remote compute , please verify by running a CI build on this branch

Reply:
Was executed, see link in PR description